### ISSUE TITLE
Fixed Stand Alone numeric convert to ZONED instead of PACKED

### ISSUE THREAD

**Issue by m-tyler** [BrianGarland/vscode-rpgfree#47]
This code is supposed to be packed data. 
Before (Production)
```
     D DDAMT#          S              9  2
     D DDGROS#         S              9  2
     D DEDAMNT         S              7  2
```
After
```
        Dcl-S DDAMT#       Zoned(9:2);
        Dcl-S DDGROS#      Zoned(9:2);
        Dcl-S DEDAMNT      Zoned(7:2);
```

Production debugger states the attribute of DDAMT# is 
![image](https://user-images.githubusercontent.com/1353665/188744971-801447aa-ec15-4cf4-b22b-4e057b873fc8.png)

**Comment by BrianGarland** [comment]
@m-tyler These get converted to packed when I try it on just those lines.   Were there other lines before these when you did it?  What version of the extension were you using?

**Comment by m-tyler** [comment]
I am currently on `v0.0.21`.  Once the conversion converts a field to zoned it always marks all numeric as zoned.  

I am looking at patterns that cause this to happen.  I'll deliver source for a test case once I find a pattern that is simple enough.

**Comment by BrianGarland** [comment]
I think I just discovered the trigger.  Once you convert a DS it looks like it things everything is zoned.

**Comment by m-tyler** [comment]
Perfect timing.  I was just about to send you a test demonstrating the same thing!

**Comment by BrianGarland** [comment]
will be fixed in 0.0.22
